Content.Server/GameObjects/Components/Interactable/Tools/CrowbarComponent.cs

```diff
@@ -27,6 +30,16 @@ public void AfterAttack(AfterAttackEventArgs eventArgs)
         {
             var mapGrid = _mapManager.GetGrid(eventArgs.ClickLocation.GridID);
             var tile = mapGrid.GetTileRef(eventArgs.ClickLocation);
+
+            var ownerLocation = Owner.Transform.GridPosition;
+            var tileCenterX = tile.X + 0.5;
+            var tileCenterY = tile.Y + 0.5;
+
+            if (Distance(ownerLocation.X, tileCenterX, ownerLocation.Y, tileCenterY) > MAX_DISTANCE)
```

Comment:
You can use `IMapGrid.GridTileToLocal` to do the work to get the `GridPosition` of the exact tile.

You can then use `GridCoordinates.Distance` to calculate the distance.